### ISSUE TITLE
shelf database problem

### ISSUE THREAD

**Issue by al-niessner** [al-niessner/DAWGIE#155]
From user:

```
Cerberus run ID 5: it is reading transit run ID 5
Cerberus run ID 6: it is reading some old output of transit that is not transit run ID 5
Cerberus no run ID (if I don’t attribute a run ID): it is reading transit run ID 5
```

This is definitely a problem and should be verified to not occur in any database by adding it to the checks for the consistency across all databases (`Test/test_13.py`).

**Comment by al-niessner** [comment]
This may not be testable after all. Need to go back to the user and get more information like exact query because there is no transit in cerberus. Discovery while trying to decide how to test such a condition is at [load state vectors](https://github.com/al-niessner/DAWGIE/blob/ee6a4e2f3c502d35535d0af95bd72c27978f084f/Python/dawgie/db/shelf.py#L278-L304) for a private pipeline. The prev is reinitialized with each state vector which means that data loaded maybe inconsistent. Being that private pipelines are for development, the probability of incomplete state vectors being written is higher.  The resultant code for loading state vectors will then put older data into the state vector if necessary. For instance if a.b.c (state vector) wrote values d,e,f on run ID 3 then d and e only on run ID 4. If one then tried to d, e, and f using run ID 5, the algorithm will d and e from run ID 4 and f from run ID 3. This may not be the desired behavior. It also does not account for when the run ID is not specified or, at least, I cannot see how it would work out.

The loading algorithm should find the largest run ID that is the consistent across all of the values.

**Comment by al-niessner** [comment]
Should be fixed with #189
